src-cpp/KafkaConsumerImpl.cpp

```diff
@@ -187,6 +187,38 @@ RdKafka::KafkaConsumerImpl::unassign () {
 }
 
 
+RdKafka::Error *
+RdKafka::KafkaConsumerImpl::incremental_assign (const std::vector<TopicPartition*> &partitions) {
+  rd_kafka_topic_partition_list_t *c_parts;
```

Comment:
Since this is C++ and not ancient C89 we don't need to put variable definitions at the top of the scope but where they are used 🎉 :)

(no action required, just wanted to point it out)

Reply:
yeah, this is primarily due to just following the pattern set elsewhere in this class.